pycaret/internal/plots/utils/time_series.py

```diff
@@ -24,6 +25,46 @@
 PlotReturnType = Tuple[Optional[go.Figure], Optional[Dict[str, Any]]]
 
 
+#### Data Types allowed for each plot type ----
+# First one in the list is the default (if requested is None)
+ALLOWED_PLOT_DATA_TYPES = {
```

Comment:
Maybe we can use enums here?

Reply:
ok, makes sense. Maybe I will add that as a final refactor to make it easier since I am in the middle of more changes. Hope that is fine.

Reply:
I opened another issue for this for tracking. Does this only apply to ALLOWED_PLOT_DATA_TYPES? 
Also, are you ok merging this if we do this refactor later?